### ISSUE TITLE
Scheduler Documentation

### ISSUE THREAD

**Issue by Daparrag** [jigangkim/nvidia-gpu-scheduler#1]
Hi Jigan,

I would like how in priniple your scheduler works. Do you have some documentation.

**Comment by jigangkim** [comment]
README.md is the only documentation I have for now..

The basic principle of the scheduler is running multiple NVIDIA GPU processes (e.g. multiple tensorflow instances for hyperparameter tuning) with python multiprocessing.
It has basic load balancing functionality (for multi-GPU machines) and job scheduling via py3nvml (https://pypi.org/project/py3nvml/).

If you are interested in using the package, you can take a look at the examples (example.py, example_openaibaselines.py).
The short answer is you need to define the **child_process()** (some GPU application you want to run) function and **child_process_args()** (converting json configuration file to child process args) function + provide **a folder containing the config files** (child process args).

FYI, log_tqdm is needed to display the progress bars but is NOT necessary to run the program.

**Comment by jigangkim** [comment]
> Hi Jigan,
> 
> I would like how in priniple your scheduler works. Do you have some documentation.
